src/v2/auth.rs

```diff
@@ -89,6 +85,16 @@ impl Client {
     ///
     /// On success, the returned token will be valid for all requested scopes.
     pub fn login(&self, scopes: &[&str]) -> FutureTokenAuth {
+        // TODO
+        trace!("login called");
+
+        self.login_hyper(scopes)
+    }
+
+    /// Perform registry authentication and return an authenticated token.
+    ///
+    /// On success, the returned token will be valid for all requested scopes.
+    pub fn login_hyper(&self, scopes: &[&str]) -> FutureTokenAuth {
```

Comment:
This can stay private or `pub(crate)`.

Reply:
I'll remove the indirection for now as I haven't implemented the login in reqwest after all.